ci-operator/step-registry/ipi/conf/osimage/nutanix/ipi-conf-osimage-nutanix-commands.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z ${CLUSTER_OS_IMAGE} ]]; then
+    echo "Get default clusterOSImage"
+    CLUSTER_OS_IMAGE=$(openshift-install coreos print-stream-json | jq -r '.architectures.x86_64.artifacts.nutanix.formats.qcow2.disk.location')
```

Comment:
Maybe you need to use python command to load the json file to workaround `jq` missing issue in installer image, something like https://github.com/openshift/release/blob/e747ed8f27f111024a2dc3755b2d7ae5a8592b45/ci-operator/step-registry/aws-load-balancer/pre-install-rosa/aws-load-balancer-pre-install-rosa-commands.sh#L75

But one more concern is if we always use the default boot image from installer to customize the cluster OS image, I did not see the difference compared with the case that we did not set any value for cluster OS image. I think installer always use the same way to get the OS image to boot the cluster installation when no any customization setting.

Reply:
I'm trying to cover the clusterOSImage field in install config, the logic is different by setting it or not, the value of it doesn't matter to much. Or what would you suggest, maintain an httpd server for downloading it? is there existing ones?

Reply:
>  the logic is different by setting it or not, the value of it doesn't matter to much

If not set, what is the default os image? From my understanding, if not setting, installer would going to download the same version of disk as the output of `openshift-install coreos print-stream-json`.

> Or what would you suggest, maintain an httpd server for downloading it.

I guess you are focus on testing where to download the image, but actually I am concerning what version of disk is being downloading. Just like what I explained, `openshift-install coreos print-stream-json` would print out different version of disk image by different version of installer, that means, the disk image version vary time by time. If you want to test `clusterOSImage`, I would suggest always need to set `CLUSTER_OS_IMAGE` on the job when the step is referenced, but never feed a default value. E.g: get a disk image url from  `openshift-install coreos print-stream-json` using an older version of installer.